src/h5web/visualizations/shared/AxisSystem.tsx

```diff
@@ -62,16 +68,28 @@ function AxisSystem(props: Props): JSX.Element {
   });
 
   // Restrain ticks scales to visible domains
-  const xTicksScale = abscissaInfo.scaleFn();
+  const xTicksScale = SCALE_FUNCTIONS[
+    abscissaConfig.scaleType || ScaleType.Linear
+  ]();
   xTicksScale.domain(visibleDomains[0]);
   xTicksScale.range([0, width]);
 
-  const yTicksScale = ordinateInfo.scaleFn();
+  const yTicksScale = SCALE_FUNCTIONS[
+    ordinateConfig.scaleType || ScaleType.Linear
+  ]();
```

Comment:
It is the only strong drawback that I encountered: as `scaleType` can be undefined in `AxisConfig`, I have to supply a fallback it here.

In other places (`Tooltip`, `LineVis`...), the fallback is handled in `getAxisScale`.

Reply:
`scaleType` is now always in `AxisConfig`